CleffyNotes/ViewControllers/FeaturesViewController.swift

```diff
@@ -57,25 +64,29 @@ class FeaturesViewController: UIViewController {
     func setTheme() {
         transposition.layer.cornerRadius = 8
         keysAndScales.layer.cornerRadius = 8
+        metronome.layer.cornerRadius = 8
         changeThemeButton.layer.cornerRadius = 8
         if let theme = themeHelper.themePreference {
             switch theme {
             case "Green":
                 view.backgroundColor = .myLightGreen
                 transposition.backgroundColor = .myGreen
                 keysAndScales.backgroundColor = .myGreen
+                metronome.backgroundColor = .myGreen
                 changeThemeButton.backgroundColor = .myGreen
                 navigationController?.navigationBar.barTintColor = .myGreen
             case "Blue":
                 view.backgroundColor = .myLightBlue
                 transposition.backgroundColor = .myBlue
                 keysAndScales.backgroundColor = .myBlue
+                metronome.backgroundColor = .myBlue
                 changeThemeButton.backgroundColor = .myBlue
                 navigationController?.navigationBar.barTintColor = .myBlue
             default:
                 view.backgroundColor = .myLightPurple
                 transposition.backgroundColor = .myPurple
                 keysAndScales.backgroundColor = .myPurple
+                metronome.backgroundColor = .myPurple
                 changeThemeButton.backgroundColor = .myPurple
                 navigationController?.navigationBar.barTintColor = .myPurple
             }
```

Comment:
This could possibly be simplified by creating a temporary variable, setting it to the correct color, and then using that color to tint everything.